_modules/hubble.py

```diff
@@ -218,6 +218,11 @@ def audit(configs=None,
     if not called_from_top and not results:
         results['Messages'] = 'No audits matched this host in the specified profiles.'
 
+    for error in ret.get('Error', []):
+      if not terse_results.has_key('Error'):
```

Comment:
Shouldn't this be `if not results.has_key('Error'):` ? I think you might overwrite all but the last error with the code as it is.